linked-list-dbl/test_doubly_linked_list.py

```diff
@@ -0,0 +1,54 @@
+from doubly_linked_list import Doublylinkedlist
+import pytest
+
+
+@pytest.fixture()
+def list_setup():
+    dbl_list = Doublylinkedlist()
+    dbl_list.insert(1)
+    dbl_list.insert(2)
+    dbl_list.insert(3)
+    dbl_list.insert(4)
```

Comment:
A fun way to save a couple lines would be something like this 
```
for i in range(1, 5):
    dbl_list.insert(i)
```
Its also a cool way to load test it with a massive number.